webpack.config.js

```diff
@@ -69,28 +95,8 @@ const blockEditor = {
 		'amp-block-editor': './assets/src/block-editor/index.js',
 		'amp-block-validation': './assets/src/block-validation/index.js',
 	},
-	module: {
```

Comment:
I've decided to use to the default CSS module loader rules. One effect of that change is that CSS file names will no longer have the name `[name]-compiled.css`, but simply `[name].css`. I've also updated the sections of code to reference the new file names.